src/content-blocks/BlockProjectsSpotlight/BlockProjectsSpotlight.scss

```diff
@@ -0,0 +1,47 @@
+@import '../../styles/settings/colors';
+
+.o-projects-spotlight {
```

Comment:
Styling is al beschikbaar van Mono: `avo2-prototype/content/scss/_c-projects-grid.scss`

Reply:
😭 mijn mooie grid css